src/__test__/components/data-exploration/differential-expression-tool/AdvancedFilteringModal.test.jsx

```diff
@@ -55,4 +64,16 @@ describe('Advanced filtering modal', () => {
     expect(screen.getByText('logFC')).toBeInTheDocument();
     expect(screen.getByPlaceholderText('Insert value').value).toEqual('0.0');
   });
+
+  it('Clicking launch the differential expression', async () => {
```

Comment:
Could you add some separation between the lines acccording to what they are doing adn perhaps some comments explaining the checks? Nothing too fancy, a little bit like in `CellSetsTool.test.jsx`, it just makes the tests way more readable.